src/devices/src/virtio/net/device.rs

```diff
@@ -78,16 +80,16 @@ pub struct Net {
     tx_iovec: Vec<(GuestAddress, usize)>,
     tx_frame_buf: [u8; MAX_BUFFER_SIZE],
 
-    interrupt_status: Arc<AtomicUsize>,
+    pub(crate) interrupt_status: Arc<AtomicUsize>,
     interrupt_evt: EventFd,
 
-    config_space: Vec<u8>,
+    pub(crate) config_space: Vec<u8>,
```

Comment:
nit: we can replace the Vec with structure for which we implement ByteValued.

Reply:
I added another commit in which I did this. Is this what you had in mind?